scripts/report/_01_make_reports.py

```diff
@@ -164,13 +166,43 @@ def plot_decoding_scores_gavg(cfg, decoding_data):
     ci_lower = decoding_data['mean_ci_lower'].squeeze()
     ci_upper = decoding_data['mean_ci_upper'].squeeze()
     metric = cfg.decoding_metric
+    clusters = np.atleast_1d(decoding_data['clusters'].squeeze())
 
     fig, ax = plt.subplots()
+    ax.set_ylim((-0.025, 1.025))
+
+    # Start with plotting the significant time periods according to the
+    # cluster-based permutation test
+    n_significant_clusters_plotted = 0
+    for cluster in clusters:
+        cluster_times = cluster['times'][0][0].squeeze()
+        cluster_p = np.asscalar(cluster['p_value'][0][0])
+        if cluster_p >= cfg.cluster_permutation_p_threshold:
+            continue
+
+        # Only add the label once
+        if n_significant_clusters_plotted == 0:
+            label = (f'$p$ < {cfg.cluster_permutation_p_threshold} '
+                     f'(cluster pemutation)')
+        else:
+            label = None
+
+        ax.fill_betweenx(
+            y=ax.get_ylim(),
+            x1=cluster_times[0],
+            x2=cluster_times[-1],
+            facecolor='orange',
+            alpha=0.15,
+            label=label
```

Comment:
looking at https://output.circle-artifacts.com/output/job/0434ff17-cd42-41c9-9169-1910efee7cfc/artifacts/0/reports/ERP_CORE/sub-average_ses-N400_task-N400_report.html#global10 I see that two clusters are displayed in the legend. Can we have only one?

Reply:
Sorry, the example link I shared with you is slightly outdated. This problem has meanwhile been fixed